app/routes/teams.rb

```diff
@@ -163,20 +164,37 @@ class Teams < Core
       end
 
       # Accept an invitation to join a team.
-      put '/teams/:slug/confirm' do |slug|
+      put '/teams/:slug/accept_invite' do |slug|
         please_login
         only_with_existing_team(slug) do |team|
-          unless team.unconfirmed_members.include?(current_user)
+          unless team.member_invites.include?(current_user)
             flash[:error] = "You don't have a pending invitation to this team."
             redirect "/"
           end
 
-          team.confirm(current_user.username)
+          team.accept_invite(current_user.username)
 
+          flash[:success] = "You are now a member of team #{slug}."
           redirect "/teams/#{slug}"
         end
       end
 
+      # Refuse an invitation to join a team.
+      put '/teams/:slug/refuse_invite' do |slug|
```

Comment:
I was just wondering the same thing, looking at `accept_invite` above.

I think we might be dancing around the concept of a `/teams/:slug/invitations` resource.

Creating the invitation would be a... `PUT`? `POST`?

To accept the invitation would probably be a `POST`, perhaps with a form value in the post body. To reject it would be `DELETE`?

Actually, it would probably be clearer to use something like this:

```
POST /teams/:slug/invitation # invite someone
POST /teams/:slug/invitation/accept
POST /teams/:slug/invitation/reject
```